lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -103,6 +104,8 @@ func (s *builderServicer) Build(ctx context.Context, request *builder_protos.Bui
 	gwRan := cellularGwConfig.Ran
 	gwEpc := cellularGwConfig.Epc
 	gwNonEpsService := cellularGwConfig.NonEpsService
+	// gwKey is a unique string for each gateway since hardware ID is not accessible here
+	gwKey := network.ID + cellGW.Type + cellGW.Key
```

Comment:
@hcgatewood Is this okay to use as a unique key for a gateway since the hardware ID of the gateway is not available? Randomization of the sync interval will be based on this key.

Reply:
couldn't we randomize the sync interval by just generating a random delay using unix time as the seed? my understanding is this was to prevent a storm of traffic, so an occasional collision probably isn't the end of the world

Reply:
We want to make the randomization deterministic for each gateway. The original PR (#7513) does it this way as well probably to avoid config updates on gateways.

Reply:
got it, hadn't seen the first pr but looks like this was already hashed out (pun intended :))

Reply:
Added #7513 in summary as context.

Reply:
You could conceivably pull the hwid from configurator, but it's not worth the call. Also, we don't actually need this to be globally unique (or really that unique at all, just "unique enough") -- so can just use cellGW.Key as the key

Reply:
Using `cellGW.Key`